internal/dbutil/nulltime.go

```diff
@@ -32,7 +33,7 @@ func (nt *NullTime) Scan(value interface{}) error {
 		return nil
 	}
 
-	times, err := time.Parse(sqliteTimeLayout, string(date))
+	times, err := parseSqliteTimeString(string(date))
 	if err != nil {
 		return nil
 	}
```

Comment:
It's internal pkg, so I think error wrapping can be handled by caller

Reply:
ok. but the error isn't returned at all. or am i missing the intent here?

Reply:
error here(time parsing part) should never happen
otherwise the value provided is `nil`

But you are right that case any other then described above should return an error